namelayer-spigot/pom.xml

```diff
@@ -25,21 +25,15 @@
     <dependency>
       <groupId>org.spigotmc</groupId>
       <artifactId>spigot-api</artifactId>
-      <version>1.12.2-R0.1-SNAPSHOT</version>
+      <version>1.12-R0.1-SNAPSHOT</version>
```

Comment:
is 1.12 vs. 1.12.2 a hard requirement for civclassic? otherwise there is no good reason to roll back target to an older release. 1.12.2 is the release supported by NameLayer mainline right now.

Reply:
No, that should be 1.12.2